### ISSUE TITLE
add hwr (handwriting recognition)

### ISSUE THREAD

**Issue by ddvk** [juruen/rmapi#117]
i've played with the myScript api and i think i could add support for it (generation of plain text,  svg or latex and math equation recognition). 

they are offering 2000 free api calls, so if one wants to use they should register and provide their api keys vie e.g. environment variables.

it could be an additional param or a separate executable, as it only need the .zip file with the content.

Is there any interest?

**Comment by AaronDavidSchneider** [comment]
I think this would be a very nice thing! However for the math support I suppose (at least with my handwriting) that it's much faster to tex those equations (if you are a bit experienced with latex) then the whole process of conversion and correction.
I tested the demo at https://webdemo.myscript.com/views/math/index.html and it seems that it has  some problems. 

![Screenshot 2020-07-01 at 13 00 49](https://user-images.githubusercontent.com/7556347/86236771-e89a3780-bb9a-11ea-826e-8eb62249d867.png)
This should correspond to:
```Latex
\begin{equation}
  \frac{\partial \rho}{\partial t}\rho + \frac{\partial}{\partial x} \left(\rho v \right) = 0
\end{equation}
```

![Screenshot 2020-07-01 at 12 58 29](https://user-images.githubusercontent.com/7556347/86236779-eb952800-bb9a-11ea-97e3-cc175716f9c0.png)
This should correspond to:
```Latex
\begin{equation}
  \Omega_\mathrm{K} = \sqrt{\frac{GM}{a^3}}
\end{equation}
```

**Comment by ddvk** [comment]
here is working prototype: https://github.com/ddvk/rmapi-hwr

i'll integrate it when/if i clean the code

**Comment by juruen** [comment]
Agreed, I think this'd be a really cool feature! As this'd be an opt-in thing, I don't see any harm in giving it a go.
